patterns/homepage.php

```diff
@@ -0,0 +1,230 @@
+<?php
+/**
+ * Title: Homepage
+ * Slug: course/homepage
+ * Inserter: yes
+ * Categories: sensei-lms
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"className":"course-homepage-pattern-parent-group","layout":{"inherit":false,"wideSize":"","contentSize":""}} -->
```

Comment:
1) We should split this pattern into small parts to be reused on the other screens.
My suggestion is
- Grow your writing skills with the course is a pattern
- Course list 
- Blog Post
-The ` Keep track of the latest news and lessons. Every week in your inbox.`
-Testimonial

We have a lot of screen using the same blocks that we should not rewrite the same instructions

Reply:
@Automattic/theam Any thoughts on this, or on this PR in general?

Reply:
Thanks for the suggestion @gabrielcaires , I've broken it into reusable smaller patterns here https://github.com/Automattic/sensei-theme/pull/44/commits/0bf443111747fd27d9bc2a24346d753de739cc03, can you take a look?